static/etc/system-image/writable-paths

```diff
@@ -12,7 +12,7 @@
 # snappy security policy, etc
 /var/lib/snapd                         auto                     persistent  transition  none
 # snap catalogue (sections, pkgnames)
-/var/cache/snapd                        auto                    persistent  transition  none
+/var/cache                              auto                    persistent  transition  none
```

Comment:
The expected behavior here on upgrade of the core18 snap is that both /writable/system-data/var/cache/snapd and /writable/system-data/var/cache/apparmor will already exist, and therefore any other contents of /var/cache on the read-only squashfs will be ignored; and /writable/system-data/var/cache will be mounted to /var/cache instead of mounting the individual subdirectories.

This looks reasonable and acceptable to me, as the only other contents of /var/cache in the current core18 snap are /var/cache/ldconfig (readable only by root and unused at runtime) and /var/cache/fontconfig (an empty directory).

Reply:
Thanks for bring the upgrading concern, let's keep it to see if @mvo5 has any feedback to this.